packages/api/cms-api/src/dam/files/file-image.resolver.ts

```diff
@@ -1,10 +1,12 @@
 import { Args, Int, Parent, ResolveField, Resolver } from "@nestjs/graphql";
 
+import { RequiredPermission } from "../../user-permissions/decorators/required-permission.decorator";
 import { ImagesService } from "../images/images.service";
 import { FileImage } from "./entities/file-image.entity";
 import { FilesService } from "./files.service";
 
 @Resolver(() => FileImage)
+@RequiredPermission(["pageTree", "dam"])
```

Comment:
Why do I need the page tree here? There may be scenarios where a user could use images in structured content

Reply:
Actually, the required permission is an OR-logic, so either "pageTree" or "dam"-permissions are necessary. So if you have "pageTree"-permission you also have access to "dam". Nevertheless only "dam" should be perfectly fine here: https://github.com/vivid-planet/comet/pull/1223/commits/59e9d5f72ab597d09fc7198d3b9ae28f9781d17e